webapp/publisher/snaps/logic.py

```diff
@@ -310,3 +328,25 @@ def convert_date(date_to_convert):
     """
     date_parsed = parser.parse(date_to_convert).replace(tzinfo=None)
     return date_parsed.strftime("%B %Y")
+
+
+def categorise_media(media):
```

Comment:
nice one, but could this be refactored to make one iteration on the array media instead?